smoothie-androidx/build.gradle

```diff
@@ -28,9 +28,9 @@ dependencies {
   compileOnly project(':toothpick')
   api project(':smoothie')
   compileOnly deps.inject
-  compileOnly deps.androidxv4
+  compileOnly deps.androidxv7
 
-  testImplementation deps.androidxv4
+  testImplementation deps.androidxv7
```

Comment:
isn't `deps.fragment` enough on these 2 lines?

Reply:
Yes, it will be enough